src/Admin/HomeBlockAdmin.php

```diff
@@ -89,6 +90,24 @@ protected function configureFormFields(FormMapper $formMapper)
                 ])
             ;
         }
+
+        if (HomeBlock::TYPE_BANNER === $this->getSubject()->getType()) {
+            $formMapper
+                ->add('titleCta', null, [
+                    'label' => 'Texte du CTA',
+                    'required' => false,
+                ])
+                ->add('colorCta', ChoiceType::class, [
```

Comment:
@tgalopin I have a list of colors (specified in the ticket), and more it matches the class names. Do you really prefer to use text input ?